### DISCUSSION TITLE
[Urgent]Charset used for the report type "GET_AMAZON_FULFILLED_SHIPMENTS_DATA_GENERAL"

### DISCUSSION THREAD

**Discussion by yasunori0120** [saleweaver/python-amazon-sp-api#202] [Q&A]
Hi,

I wrote a code using python to get a report of the report type "GET_AMAZON_FULFILLED_SHIPMENTS_DATA_GENERAL", and obtained it with "get_report_document(document_id=documentID, decrypt=True)".

However, Japanese words in the report were totally garbled and I can't fix this problem.
Does anyone know how to fix it?
With what charset the report is encoded?

A prompt reply would be greatly appreciated. This issue is very URGENT.

Thanks,
Yasunori (from Japan)

**Comment by unstory** [comment]
I have the same problem.How to solve this problem? Thanks a lot.
